src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -1357,22 +1371,59 @@ function optimal_co2_ratio_c3(
     return χ, ξ, mj, mc
 end
 
+function optimal_ξ_c3(Kmm::FT, Γstar::FT, ηstar::FT, β::FT, Drel::FT) where {FT}
+    return sqrt(β * (Kmm + Γstar) / (Drel * ηstar))
+end
+
+function compute_mj(ξ::FT, Γstar::FT, ca::FT, VPD::FT) where {FT}
+    γ = Γstar / ca
+    χ = Γstar / ca + (1 - Γstar / ca) * ξ / (ξ + sqrt(VPD))
+    return (χ - γ) / (χ + 2 * γ)
+end
+
+function compute_mc(ξ::FT, Kmm::FT, Γstar::FT, ca::FT) where {FT}
+    γ = Γstar / ca
+    κ = Kmm / ca
+    χ = Γstar / ca + (1 - Γstar / ca) * ξ / (ξ + sqrt(VPD))
+    return (χ - γ) / (χ + κ)
+end
+
+function compute_ci(ξ::FT, ca::FT, Γstar::FT, VPD::FT) where {FT}
+    return (ca * ξ + Γstar * sqrt(VPD)) / (ξ + sqrt(VPD))
+end
 
 """
-    pmodel_gs(
+    pmodel_gs_co2(
         χ::FT, 
         ca::FT,
         A::FT
     ) where {FT}
 
-    Computes the stomatal conductance of CO2 (`gs`), in units of mol CO2/m^2/s
+    Computes the stomatal conductance of CO2 (`gs_co2`), in units of mol CO2/m^2/s
     via Fick's law. Parameters are the ratio of intercellular to ambient CO2 
-    concentration (`χ`), the ambient CO2 concentration (`ca`), and the 
-    assimilation rate (`A`). This is related to the conductance of H2O by a 
+    concentration (`χ`), the ambient CO2 concentration (`ca`, in mol/mol), and the 
+    assimilation rate (`A`, mol m^-2 s^-1). This is related to the conductance of water by a 
     factor Drel = 1.6. 
```

Comment:
```suggestion
    factor Drel (with default value 1.6). 
```
since the value of Drel can be overriden